jodd-mail/src/main/java/jodd/mail/EmailAddress.java

```diff
@@ -467,9 +467,9 @@ private InternetAddress pullFromGroups(Matcher m) {
 	private String[] _calcMatcherParts(Matcher m) {
 		String currentLocalpart = null;
 		String currentDomainpart = null;
-		String localPartDa = null;
+		String localPartDa;
 		String localPartQs = null;
-		String domainPartDa = null;
+		String domainPartDa;
 		String domainPartDl = null;
 		String personal_string = null;
```

Comment:
So? SonarCube is a mere tool. It has its own limitation, and can't think for the developer.